mkdocs.yml

```diff
@@ -12,6 +12,7 @@ nav:
       - AI Apps: users/ai-apps.md
       - How To: users/howto.md
       - Support: users/support.md
+      - Remote and Local pairing: developers/edge-connection.md
```

Comment:
@BKristenssonAlfsson I see the file is now referenced in the User section, which makes sense, but it resides in the developer directory. Is that the intention?